### ISSUE TITLE
Installation instructions?

### ISSUE THREAD

**Issue by TheNomad11** [shonebinu/flashify#1]
Hi I was just searching github for flashcard web applications and found yours and tried to install it without luck (no stylesheets loading, registering not possible etc). I just uploaded it to a webspace, apllied permissions, added database details into secrets.php but that was not enough. Thanks!

Edit: Some paths in the header of index.php are wrong `/styles/...` insteas of `styles/...`

**Comment by shonebinu** [comment]
Hi, thanks for your interest in this project.

This is a LAMP project and it might be hard to run on your own now. This is something that I'm working on my own for my Uni mini-project, it's only 10% complete. I've a long way to go and I'm planning to publish it in my server, but it would still at the best a "mini-project", dunno if it will be enough to serve your purpose.

Are you trying to use this for your studies? Then maybe things like Anki, Quizlet would be a better option.

**Comment by TheNomad11** [comment]
Thanks for your answer. I know Anki and Quizlet but I would prefer something selfhosted and web based that also friends can use. LAMP sounds easy to run? Like Wordpress? I have tried several selfhosted options, I wasn't convinced by most of them. Good luck with your project, I am looking forward to updates!

**Comment by shonebinu** [comment]
Hello, completed the prototype and also added instructions on running locally... Mind you that, as i said before, this is a mini project so please don't expect anything big

**Comment by TheNomad11** [comment]
Very nice! Looks like a good start for something bigger. In this field we need more projects like yours!
Installing via docker didn't work, so I installed it the traditional way as you described. Worked fine.
The docker logs said something like that the index.php file was missing, so I got permission denied, and on Firefox the Decks cannot be seen, while on Vivaldi it looks fine 
![screen-2024-10-17-21-41-27](https://github.com/user-attachments/assets/797437e1-d38b-48dd-9a1d-bf8b1a31c600)
